subsys/bluetooth/host/hci_core.h

```diff
@@ -84,6 +79,22 @@ enum {
 	 * events, or both.
 	 */
 	BT_ADV_LIMITED,
+	/* Advertiser set is currently advertising in the controller. */
+	BT_ADV_ADVERTISING,
+	/* Advertiser should include name in advertising data */
+	BT_ADV_ADVERTISE_NAME,
+	/* Advertiser set is connectable */
+	BT_ADV_CONNECTABLE,
+	/* Advertiser set has disabled the use of private addresses and is using
+	 * the identity address instead.
+	 */
+	BT_ADV_USE_IDENTITY,
+	/* Advertiser has been configured to keep advertising after a connection
+	 * has been established as long as there are connections available.
+	 */
+	BT_ADV_KEEP_ADVERTISING,
+	/* Advertiser has been temporarily disabled. */
+	BT_ADV_ADVERTISING_PAUSED,
```

Comment:
I know you've mostly just copied the existing names, but we could take the opportunity to remove the extra "ADV" redundancy here and make these e.g. `BT_ADV_ENABLED` instead of `BT_ADV_ADVERTISING`, BT_ADV_INCLUDE_NAME` instead of `BT_ADV_ADVERTISE_NAME`, `BT_ADV_PERSIST` instead of `BT_ADV_KEEP_ADVERTISING`, and `BT_ADV_PAUSED` instead of `BT_ADV_ADVERTISING_PAUSED`.

Reply:
Good suggestion. I already did it to some of them to make them shorter.